### ISSUE TITLE
Global / DAO settings

### ISSUE THREAD

**Issue by luisivan** [aragon/design#13]
I saw a mockup that @jounih designed in which the Settings app contained two tabs, one for the settings of Aragon Core, and one for the DAO settings.

I want to make a slightly different proposal that I think matches better with [the DAO switcher](https://github.com/aragon/design/issues/9).

I think we should split the two apps into the local DAO settings and the Aragon Core settings. The local DAO settings would be exactly what we have now, the last of the apps in the sidebar.

And the DAO settings would sit next to the DAO chooser and the notifications, and be a different app. We can even call it something like _Preferences_ if we worry people may confuse them.

I think this would be a step forward to make sure Aragon Core stays multi-DAO and doesn't generate confusion in terms of which apps run where (Aragon Core or this particular DAO?)

**Comment by jounih** [comment]
@luisivan good points. We actually discussed something similar with @bpierre but the other way around - having a settings icon next to the login/username area at the bottom of the left nav for global app settings, and using the settings app for DAO settings only.

The top part of the left sidebar is already crowded with DAO icon, Dao name and notifications icon - otherwise that could be a good place too. Currently we have more space at the bottom and it could be a logical place as a "user" area for "user"/global app settings

**Comment by luisivan** [comment]
Totally agree! Let's do that

**Comment by jounih** [comment]
OK after a bit of rejigging we came up with the following:

Use a smaller avatar for the DAO (a blocky ideally) and add the app settings icon at the top next to the notification icon. It didn't really work in the bottom, got too busy.

I think we can also perhaps remove the current settings app completely if we do this - the only content left on it would only be some info about the DAO we display have elsewhere - like on the Home app maybe?

![webapp-1366px - app-settings-dao-blocky 2x](https://user-images.githubusercontent.com/10109867/46335824-a720d500-c620-11e8-9bdd-1303de6dc006.jpg)

**Comment by jounih** [comment]
OK, after some though this is what we have.

Preferences relating to the web-app would be accessible from the cog icon in the bottom. That would also only leave DAO specific things in the "Settings" app, so perhaps we can rename that app "DAO info" or "Info" or "Org info"


Prototype:

[https://invis.io/MFOF866ET3N#/323903431_WebApp-1366px_-_Logged_In](https://invis.io/MFOF866ET3N#/323903431_WebApp-1366px_-_Logged_In)

Design:

![webapp-1366px - logged in 2x](https://user-images.githubusercontent.com/10109867/46526577-88bc1300-c886-11e8-9316-47077a7a46dd.jpg)
![webapp-1366px - app settings modal 2x](https://user-images.githubusercontent.com/10109867/46526576-88bc1300-c886-11e8-9338-aecea1870e60.jpg)

**Comment by sohkai** [comment]
@jounih I noticed in the mock up that the Settings app has been removed from the left menu bar. At some point I think we discussed putting the "DAO Settings" (or info) into the org switcher dropdown, is that still the plan?

**Comment by jounih** [comment]
@sohkai Yep we could totally put it into the DAO switcher, will update the issue for that with a design

**Comment by luisivan** [comment]
I think Settings should just be an app in the app bar, not the DAO switcher. Apart from that, I like the name Settings for DAO settings and Preferences for the Aragon Core settings

**Comment by jounih** [comment]
We don't have traditional "settings" right now for the DAO - it's just mainly info about the DAO - app addresses and a way to request test tokens. 

After discussing some ideas with @bpierre, I updated the DAO switcher issue with a tweaked proposal:

https://github.com/aragon/design/issues/9

**Comment by luisivan** [comment]
Cool, sounds good in the case that it's just read-only info. I'm also thinking when we will actually have DAO-wide settings, such as the Home app content or the color accent. But that may be part of the Identity app.

**Comment by jounih** [comment]
Yep let's revisit this once we have more DAO settings!

**Comment by jounih** [comment]
Added access to preferences to other situations where user might need access to them, as requested by @bpierre - throughout the onboarding journey, and on error screens. It can always stay in the same position on these type of screens - top left of viewport.

![screen1 2x](https://user-images.githubusercontent.com/10109867/47569451-90be1e80-d92b-11e8-99ef-a2fec3e3107e.jpg)
![app error 2x](https://user-images.githubusercontent.com/10109867/47569452-90be1e80-d92b-11e8-8e43-5992a687a560.jpg)

**Comment by sohkai** [comment]
@jounih @bpierre and I were discussing moving the settings to be in the bottom left corner in the onboarding and error states, so that its location is consistent throughout the app.

I was also thinking–maybe it'd be nice to show the address badge of the current account in all states too?

**Comment by dizzypaty** [comment]
We need to explore the concept of a preferences module/section that is available throughout the app and consistent in its location so we increase memorability of the interface. This section should contain the access to global preferences modal, the network connection indicator and the current account address badge.

For now, the most reasonable area to place it is the bottom left, inside the app menu. I think we should also add a label that explicitly says ‘Global preferences’ so don’t just rely on users guessing what would be behind the cog icon. 

I also feel that global preferences should open a full screen modal that has a slightly different visual treatment so users can differentiate the nature of these “global preferences” VS the “specific organization’s settings”. cc @jounih 

I will add some sketches later.

**Comment by jounih** [comment]
@dizzypaty Sounds good. Do you want to mock this up or want me to take a stab?

**Comment by dizzypaty** [comment]
@jounih Please go ahead based on what we discussed!

**Comment by jounih** [comment]
Done.

**Design in figma**

https://www.figma.com/file/QFizPRIWRn6toUp2mPfg5vRE/Global-preferences-and-custom-labels?node-id=8%3A2
